app/views/layouts/application.html.erb

```diff
@@ -101,6 +101,17 @@
   <div class="footer">
     <div class="container">
       <div class="footer_info">
+
+<ul class="footernav">
```

Comment:
They are formatted differently though, so I'm not sure of an easy way. (in the header it's two lists so I can pull "about" right, and in the header there's no copyright link)

If one of you want to take this on feel free, but I don't want it to hold up this PR